src/components/mainPage/MainPage.tsx

```diff
@@ -1,15 +1,30 @@
+import { useState } from 'react'
+
 import Footer from '../footer/Footer'
 import Header from '../header/Header'
-import Navbar from '../navbar/Navbar'
+import ModalCreateBooks from '../modal/ModalCreateBooks'
+import NavbarItems from '../navbar/navbarItems/NavbarItems'
 
 import './MainPage.css'
 
 const MainPage = () => {
+  const [ modalIsOpen, setModalIsOpen ] = useState(false)
+
+
+  const handleModalOpen = () => {
+    setModalIsOpen(false)
+  }
+
+  const handleModalClose = () => {
+    setModalIsOpen(true)
+  }
+
   return (
     <div className='main_page_container'>
-      <Header />
-      <Navbar />
-      <Footer />
+      <Header/>
+      <NavbarItems setModalIsOpen={handleModalClose}/>
+      <ModalCreateBooks open={modalIsOpen} onClose={handleModalOpen}/>
+      <Footer setModalIsOpen={handleModalClose}/>
```

Comment:
can you please explain what are you achieving with calling handleModalOpen function to children?

Reply:
I move logic in parent component.